clmm/modbackend/ccl.py

```diff
@@ -0,0 +1,164 @@
+# Functions to model halo profiles 
+
+import pyccl as ccl
+
+from .cluster_toolkit_patches import _patch_comoving_coord_cluster_toolkit_rho_m
+
+import numpy as np
+import warnings
+
+from . import func_layer
+from . func_layer import *
```

Comment:
please do not import `*`. Either import a module and use it or make specific imports

Reply:
In general that's a good advice but in this case we are using the __all__ variable to control the exports from func_layer. That way, we do not need to repeat the functions names in all back-ends, and once a new function in added to func_layer __all__ variable, all back-ends will have it automatically.